src/mame/namco/namcos10.cpp

```diff
@@ -783,6 +1006,76 @@ TIMER_DEVICE_CALLBACK_MEMBER(namcos10_state::io_update_interrupt_callback)
 	m_io_update_interrupt(1);
 }
 
+void namcos10_state::namcos10_exio(machine_config &config)
+{
+	NAMCOS10_EXIO(config, m_exio, XTAL(22'118'400));
+
+	auto m_exio_cpu = subdevice<tmp95c061_device>("exio:exio_mcu");
+	m_exio_cpu->port7_write().set([this] (uint8_t data) {
+		m_exio_analog_idx = data;
+	});
```

Comment:
Please expose this stuff in the EXIO device – you shouldn’t be reaching inside devices like this.